trapi.rkt

```diff
@@ -371,7 +374,11 @@
                                  (cons node-set unmerged-bags)
                                  rest)))))))
            node-sets
-           (append (map string->symbol (set->list (apply set-union node-sets))) curies)))
+           (append (map string->symbol
+                        (set->list (if (null? node-sets)
+                                     '()
+                                     (apply set-union node-sets))))
```

Comment:
Not sure if this is possible in your case, but if `node-sets` is the empty list, this will fail, because `set-union` requires at least one argument.  If an empty `node-sets` is possible, I would change this to:

```
(apply set-union (set) node-sets)
```

for safety.

Reply:
I didn't read carefully, you're already checking for that case.  Great.